Rego/AADConfig.rego

```diff
@@ -933,97 +1042,111 @@ tests[{
 #
 # MS.AAD.8.1v1
 #--
+
 # must hardcode the ID. See
 # https://docs.microsoft.com/en-us/azure/active-directory/enterprise-users/users-restrict-guest-permissions
 # pattern matching on function args: https://docs.styra.com/regal/rules/idiomatic/equals-pattern-matching
+# return specified string based on id passed to method.
 LevelAsString("2af84b1e-32c8-42b7-82bc-daa82404023b") := "Restricted access"
+
 LevelAsString("10dae51f-b6af-4016-8d66-8c2a99b929b3") := "Limited access"
+
 LevelAsString("a0b1b346-4d3e-4e8b-98f8-753987be4970") := "Same as member users"
-LevelAsString(Id) := "Unknown" if {not Id in [
+
```

Comment:
I can't use the constants for the LevelAsString method because of how the function works, but I can use them from line 1055 to 1068. Please confirm this is what you were thinking & resolve.